tests/engine/models/test_history_embedder.py

```diff
@@ -8,23 +8,43 @@
 )
 
 
-def test_lstm_history_embedder_proper_shape():
-    SE = 64  # service embedding dimension
-    N = 10  # length of a sequence
-    batch_size = 32
+@pytest.fixture
+def params():
+    batch_size = 64
+    max_N = 10  # length of an output sequence
+    SE = 128  # service embedding dimension
+    return batch_size, max_N, SE
+
 
-    example_history = torch.rand(batch_size, N, SE)
+def test_lstm_history_embedder_proper_shape(params):
+    batch_size, _, SE = params
+    example_history = torch.rand(batch_size, 25, SE)
     history_embedder = LSTMHistoryEmbedder(SE)
 
     assert history_embedder(example_history).shape == torch.Size([batch_size, SE])
 
 
-def test_mlp_history_embedder_proper_shape():
-    SE = 64  # service embedding dimension
-    N = 10  # length of a sequence
-    batch_size = 32
+def test_mlp_history_embedder_proper_shape(params):
+    batch_size, max_N, SE = params
+    history_embedder = MLPHistoryEmbedder(SE, max_N)
 
-    example_history = torch.rand(batch_size, N, SE)
-    history_embedder = MLPHistoryEmbedder(SE, N)
+    example_history = torch.rand(batch_size, 5, SE)
+    assert history_embedder(example_history).shape == torch.Size([batch_size, SE])
 
+    example_history = torch.rand(batch_size, 20, SE)
     assert history_embedder(example_history).shape == torch.Size([batch_size, SE])
+
+
+def test_mlp_history_embedder_align_history(params):
+    batch_size, max_N, SE = params
+    history_embedder = MLPHistoryEmbedder(SE, max_N)
+
+    history = torch.rand(batch_size, 5, SE)
```

Comment:
Maybe the history shape should be randomized? If our recommender only uses the hardcoded values from _params_ function maybe we should randomize at least the second dimension of its shape? 
I believe that it is preferable to avoid hard coding values.